lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4616,17 +4672,15 @@ LogicalResult StmtEmitter::visitSV(CoverConcurrentOp op) {
 /// `verif.assert`, `verif.assume`, and `verif.cover`.
 LogicalResult StmtEmitter::emitVerifAssertLike(Operation *op, Value property,
                                                PPExtString opName) {
-  if (hasSVAttributes(op))
-    emitError(op, "SV attributes emission is unimplemented for the op");
 
   // If we are inside a procedural region we have the option of emitting either
   // an `assert` or `assert property`. If we are in a non-procedural region,
   // e.g., the body of a module, we have to use the concurrent form `assert
   // property` (which also supports plain booleans).
   //
-  // See IEEE 1800-2017 section 16.14.5 "Using concurrent assertion statements
-  // outside procedural code" and 16.14.6 "Embedding concurrent assertions in
-  // procedural code".
+  // See IEEE 1800-2017 section 16.14.5 "Using concurrent assertion
+  // statements outside procedural code" and 16.14.6 "Embedding concurrent
+  // assertions in procedural code".
```

Comment:
Stray changes?

Reply:
probably clang-tidy

Reply:
Probably worthwhile reverting these changes. Especially the check for `hasSVAttributes` should probably stay 😄